src/nanoemoji/color_glyph.py

```diff
@@ -37,52 +37,58 @@
 import ufoLib2
 
 
-def _scale_viewbox_to_emsquare(view_box: Rect, upem: int) -> Tuple[float, float]:
-    # scale to font upem
-    return (upem / view_box.w, upem / view_box.h)
-
-
-def _shift_origin_0_0(
-    view_box: Rect, x_scale: float, y_scale: float
-) -> Tuple[float, float]:
-    # shift so origin is 0,0
-    return (-view_box.x * x_scale, -view_box.y * y_scale)
+def _scale_viewbox_to_font_metrics(
+    view_box: Rect, ascender: int, descender: int, width: int
+):
+    # scale height to (ascender - descender)
+    scale = (ascender - descender) / view_box.h
```

Comment:
suggest assert descender <= 0; conveniently makes it much more obvious that we are *adding* descender to ascender here.